### PR TITLE
[chrome-api-jstocxxtest] Add more GetStatusChange tests

### PR DESCRIPTION
**Pull request by ViktoriiaKovalova**
Add tests to check that events with various `eventState`s and `eventCount`s are handled correctly.
Add test with optional timeout parameter passed.

### CODE REVIEW

**Comment by ViktoriiaKovalova** [comment]
Looks like CI is breaking because of the github bug 
https://github.com/actions/checkout/issues/1448

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 1]
```diff
@@ -369,5 +369,106 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+  'testGetStatusChange_withTimeout': async function() {
+    const establishContextResult = EMPTY_CONTEXT_RESULT;
+    expectReportEstablishContext(
+        /*requestId=*/ 123, 'SUCCESS', establishContextResult);
+    expectReportGetStatusChange(
+        /*requestId=*/ 124, [], 'TIMEOUT');
+    mockControl.$replayAll();
+
+    launchPcscServer(/*initialDevices=*/[
+      {'id': 123, 'type': 'gemaltoPcTwinReader', 'cardType': 'cosmoId70'}
```
Comment (Fabian-Sommer) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:456]: I think considering how often the strings 'gemaltoPcTwinReader' and 'cosmoId70' are used, they deserve to be constants (throughout this file, maybe in another PR) . Especially considering that at least the Gemalto Reader already has some associated constants, and I suspect the exact string might be important.

**Review by Fabian-Sommer — APPROVED** [review]
LGTM

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 1 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js:456]: good idea! I'll make a follow-up PR :)

**Pull request merged**